src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.test.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { create } from "react-test-renderer";
+
+import BcscRedirect from "./BcscRedirect";
+
+describe("Consent Page Component", () => {
+  test("Matches the snapshot", () => {
+    const header = {
+      name: "Criminal Record Check"
+    };
+
+    const page = {
+      header
+    };
+
+    const consent = create(<BcscRedirect page={page} />);
```

Comment:
same here, change this to match the component name being tested 😄

Reply:
right